pkg/types/types.go

```diff
@@ -152,6 +158,10 @@ func (r *RekorEntry) Load(ctx context.Context) error {
 		dataReader = bytes.NewReader(r.Data)
 	}
 
+	if err := r.ParseKeys(); err != nil {
```

Comment:
I'd move this call before the URL fetch. There's no reason to even call `Get()` if we don't have a valid key/sig pair.